features/api/illumina_b/top_of_the_pipeline.feature

```diff
@@ -121,107 +121,27 @@ Feature: The top of the Illumina-B pipeline
           "transfers": {
             "A1":  "A1",
             "A2":  "A2",
```

Comment:
Surprisingly enough, this was because I accidentally fixed a bug. Previously the transition to built up a list of 'bad_wells' and subtracted them from the transition hash. However this list didn't include wells that didn't actually exist.